tests/integration/fixtures/security_report/csp_chrome_blocked_asset.no_processing.output.json

```diff
@@ -23,12 +23,18 @@
   },
   "culprit": "style-src cdn.example.com",
   "environment": "production",
+  "grouping_config": {
```

Comment:
These snapshot updates are the result of normalizing default attributes in regular normalization.